[2025-01-30T03:38:01.433+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-30T03:38:01.460+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etapa2_dag.copia manual__2025-01-30T03:32:35.404173+00:00 [queued]>
[2025-01-30T03:38:01.472+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etapa2_dag.copia manual__2025-01-30T03:32:35.404173+00:00 [queued]>
[2025-01-30T03:38:01.473+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-01-30T03:38:01.628+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): copia> on 2025-01-30 03:32:35.404173+00:00
[2025-01-30T03:38:01.637+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=382) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-30T03:38:01.639+0000] {standard_task_runner.py:72} INFO - Started process 384 to run task
[2025-01-30T03:38:01.640+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etapa2_dag', 'copia', 'manual__2025-01-30T03:32:35.404173+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/etapa2_dag.py', '--cfg-path', '/tmp/tmpk0eizbnt']
[2025-01-30T03:38:01.641+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask copia
[2025-01-30T03:38:01.698+0000] {task_command.py:467} INFO - Running <TaskInstance: etapa2_dag.copia manual__2025-01-30T03:32:35.404173+00:00 [running]> on host eed2ce10479b
[2025-01-30T03:38:01.824+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etapa2_dag' AIRFLOW_CTX_TASK_ID='copia' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T03:32:35.404173+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-30T03:32:35.404173+00:00'
[2025-01-30T03:38:01.827+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-30T03:38:01.827+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-30T03:38:01.828+0000] {logging_mixin.py:190} INFO - Current task name:copia state:running start_date:2025-01-30 03:38:01.460783+00:00
[2025-01-30T03:38:01.828+0000] {logging_mixin.py:190} INFO - Dag name:etapa2_dag and current dag run status:running
[2025-01-30T03:38:01.829+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-30T03:38:01.830+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-01-30T03:38:01.861+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-30T03:38:01.864+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etapa2_dag, task_id=copia, run_id=manual__2025-01-30T03:32:35.404173+00:00, execution_date=20250130T033235, start_date=20250130T033801, end_date=20250130T033801
[2025-01-30T03:38:01.877+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-30T03:38:01.878+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-30T03:38:01.878+0000] {logging_mixin.py:190} INFO - Dag name:etapa2_dag queued_at:2025-01-30 03:37:52.915809+00:00
[2025-01-30T03:38:01.879+0000] {logging_mixin.py:190} INFO - Task hostname:eed2ce10479b operator:PythonOperator
[2025-01-30T03:38:01.895+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-30T03:38:01.921+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-30T03:38:01.925+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
